qp/drivers/qiskit/simulator.py

```diff
@@ -0,0 +1,93 @@
+from qiskit import QuantumCircuit, execute, Aer
+
+from qp import gatearray
+
+
+gate_array_gate_to_simulator_gate_map = {
+    gatearray.H: QuantumCircuit.h,
+    gatearray.X: QuantumCircuit.x,
+    gatearray.Y: QuantumCircuit.y,
+    gatearray.Z: QuantumCircuit.z,
+    gatearray.S: QuantumCircuit.s,
+    gatearray.T: QuantumCircuit.t,
+    gatearray.Td: QuantumCircuit.tdg,
+    gatearray.CNOT: QuantumCircuit.cnot,
+    gatearray.I: QuantumCircuit.iden,
+    gatearray.SWAP: QuantumCircuit.swap,
+}
+
+
+def distribution_from_counts(counts, n_qubits):
+    n_states = n_qubits ** 2
```

Comment:
This function was a little bit difficult to understand when not having the project freshly in mind - add doc string?